pkg/api/imagereferencemutators/pods_test.go

```diff
@@ -4,163 +4,16 @@ import (
 	"reflect"
 	"testing"
 
-	kapiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-	kapi "k8s.io/kubernetes/pkg/apis/core"
 
 	_ "github.com/openshift/origin/pkg/api/install"
 )
 
-func Test_podSpecMutator_Mutate(t *testing.T) {
```

Comment:
Wasnt expecting tests to be deleted.

Reply:
That test moved to internal version (in external version we don't deal with internal objects that this test exercise)